pkg/flowkit/state_test.go

```diff
@@ -351,7 +453,12 @@ func Test_AccountByAddressSimple(t *testing.T) {
 
 	assert.Equal(t, acc.name, "emulator-account")
 }
+func Test_AccountByAddressAndNameSimple(t *testing.T) {
+	p := generateSimpleProject()
+	acc, _ := p.Accounts().ByName("emulator-account")
 
+	assert.Equal(t, acc.name, "emulator-account")
+}
```

Comment:
This looks like it's almost a duplicate of the test case below `Test_AccountByNameSimple`. Can they be combined?

`Test_AccountByAddressAndNameSimple` doesn't quite fit anymore after you renamed the method to `ByName`

Reply:
addressed